src/components/BookList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const BookList = () => {
+  const books = [
+    { id: 0, title: 'The Lord of the Rings', author: 'J.R.R. Tolkien' },
+    { id: 1, title: 'The Hobbit', author: 'J.R.R. Tolkien' },
+    { id: 2, title: 'The Silmarillion', author: 'J.R.R. Tolkien' },
+  ];
+
+  return (
+    <div>
+      <h1>Book List</h1>
+      <ul>
+        {books.map((book) => (
+          <li key={book.id}>
+            <h2>{book.title}</h2>
+            <p>{book.author}</p>
```

Comment:
This is one way of doing this task , however for React good practice you should create a <Book > component that takes the book data as a prop and renders the title and the auth
You could write for example

```bash
// BookItem.js
const BookItem = ({ book}) => (
  <li>
    {book.title}
    {' '}
    -
    {book.author}
    <button>
      Remove
    </button>
  </li>
);

// BookList.js
const BookList = ({ books}) => (
  <ul>
    {books.map((book) => (
      <BookItem key={book.id} book={book}  />
    ))}
  </ul>
);
```